README.md

```diff
@@ -20,7 +19,91 @@ That's it! All other dependencies are included in the Docker containers that dka
 ```bash
 ln -s /my/dktl/location/bin/dktl.sh ~/bin/dktl
 ```
-3. Go to the directory where DKAN-tools was downloaded to (it should contain a composer.json file) and run `composer install`. 
+
+## Starting a project
+To start a project with ``dktl`` simply create a directory.
+
+```mkdir my_project && cd my_project```
+
+Inside the project directory start the tool and initialize your project.
+
+```dktl dc up -d && dktl init```
+
+After initialization, we want to get DKAN ready.
+
+```dktl dkan:get <version_number> && dktl dkan:make```
+
+Versions of DKAN look like this: ``7.x-1.15.3``. You can see all of [DKAN's releases](https://github.com/getDkan/dkan/releases) in Github.
+
+The last prepartion step is to get Drupal.
+
+```dktl drupal:make```
+
+Finally, lets install DKAN.
+
+```dktl dkan:install```
+
+To access you site use ``dktl drush:uli``
+
+## Existing DKAN Site to DKTL
+
+One of the many reasons for using DKTL is to create a clear separation between **our** application, and other software that we are simply using. To accomplish this, we want as much of what makes our application unique to live in the ``src`` directory.
+
+To get started we find what version of DKAN our current application and run the following commands:
```

Comment:
To get started we find what version of DKAN our current application is using, and run the following commands: